lib/features/chat/domain/helix_get_channel_chat_badges.dart

```diff
@@ -0,0 +1,39 @@
+import 'package:freezed_annotation/freezed_annotation.dart';
```

Comment:
Linter rule: prefer_double_quotes (Do you have flutter/dart vscode extension installed?)
Run `flutter dart analyze .` to manually check lint rules.

Also the project style uses 120 max width, if you can add `"dart.lineLength": 120,` to your workspace settings, if not its fine.